UPGRADING.INTERNALS

```diff
@@ -79,6 +79,10 @@ PHP 8.3 INTERNALS UPGRADE NOTES
    - The function pcre_get_compiled_regex_ex has been removed.
    Use pcre_get_compiled_regex instead.
 
+ e. ext/spl
+   - The PHPAPI spl_iterator_apply() function now returns zend_result instead of int.
+     There are no functional changes.
```

Comment:
If there are no functional changes, does this require a mention in the UPGRADING notes?

Reply:
I don't know. I thought it was best to document it since it seems to be a public API. If it shouldn't be documenten I can drop this change.